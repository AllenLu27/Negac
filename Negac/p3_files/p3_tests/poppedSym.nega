int fn(){
	if (false){
		int c;
	}	
	c = 1;
}
