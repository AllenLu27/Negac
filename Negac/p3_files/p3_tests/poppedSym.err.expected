FATAL [5,2]: Undeclared identifier
Semantic analysis failed
